pandas/tests/indexes/multi/test_constructor.py

```diff
@@ -107,6 +107,13 @@ def test_copy_in_constructor():
     assert mi.levels[0][0] == val
 
 
+def test_constructor_bad_codes():
```

Comment:
bad -> invalid

add the issue number as a comment

Reply:
Done.